elements/rh-cta/rh-cta.css

```diff
@@ -42,14 +42,13 @@
   border-color: var(--rh-cta-border-color, transparent);
   border-radius: var(--rh-border-radius-default, 3px);
   border-width: var(--rh-border-width-sm, 1px);
+  gap: var(--rh-space-md, 8px);
 
   --_context-background-color: var(--rh-cta-background-color) !important;
   --_arrow-size: 13px;
 }
 
-#container pfe-icon {
```

Comment:
The actual bug was here, a usage of `pfe-` vs `pf-`.  I assumed this was a good opportunity to move to use `gap` on the flex container instead.